src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -59,6 +59,7 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit {
     return this._isExpanded;
   }
 
+  @Output() public isSelectedChange = new EventEmitter<boolean>();
```

Comment:
Last item, then we can wrap this up!

Could you please complete this emitter in the `ngOnDestroy()` method to prevent any memory leaks?

Reply:
Yup, updated it to complete the emitter